akka-http-core/src/main/scala/akka/http/impl/engine/parsing/HttpRequestParser.scala

```diff
@@ -177,6 +179,9 @@ private[http] class HttpRequestParser(_settings: ParserSettings,
             parseFixedLengthBody(contentLength, closeAfterResponseCompletion)(input, bodyStart)
           }
 
+        case Some(_) if !method.isEntityAccepted ⇒
```

Comment:
FWIW Is a `Transfer-Encoding` header generally forbidden for those methods? It seems, it is by this sentence [from RFC 7230](http://tools.ietf.org/html/rfc7230#section-3.3):

> The presence of a message body in a request is signaled by a Content-Length or Transfer-Encoding header field.